bftengine/src/preprocessor/messages/PreProcessResultMsg.cpp

```diff
@@ -27,6 +27,7 @@ PreProcessResultMsg::PreProcessResultMsg(NodeIdType sender,
                                          uint64_t reqTimeoutMilli,
                                          const std::string& cid,
                                          const concordUtils::SpanContext& spanContext,
+                                         const std::string& participant_id,
```

Comment:
Is it required to be in this message?

Reply:
Removed dependency on ResultMsg as it may not be required